### ISSUE TITLE
Compatibility with Xamarin Forms 4.0

### ISSUE THREAD

**Issue by ClausElmann** [jsmarcus/Iconize#156]
Is there a forecast to update the plugin to be compatible with Xamarin Forms 4.0?

Thanks in advance :-)

**Comment by Kilowhisky** [comment]
I have no problem with Xamarin 4.0. Works fine for me with just FA icons!

**Comment by ricavir11** [comment]
@Kilowhisky how did you configure your project to make it work ? Did you add font files to android project ?

**Comment by flamecrow** [comment]
> I have no problem with Xamarin 4.0. Works fine for me with just FA icons!

You have a sample repo of it working?

**Comment by jhermsen** [comment]
Also having compatibility issues and did some testing with different versions.
I tried the Iconize.Sample project from this github repo.
Findings:

- Compiled as is, all icons show (Xamarin.Forms 3.5.0.129452 & Iconize packages 3.5.0).
- Upgraded Xamarin.Forms to 4.1.0.581479 all icons still show.

- Upgraded all Iconize packages to 3.5.0.114 all icons still show

- Upgraded all Iconize packages to latest 3.5.0.115 all icons **not** showing.

So it stopped working at Iconize version 3.5.0.115.

**Comment by danielmeza** [comment]
@jhermsen To me is the same, I downgrade to 3.5.0.114 and works again!

**Comment by dorisoy** [comment]
Indeed, under Iconize 3.5.0.129 / Xamarin. Forms 4.2.0.709249, only FontAwesome Regular Module is valid.

> Also having compatibility issues and did some testing with different versions.
> I tried the Iconize.Sample project from this github repo.
> Findings:
> 
> * Compiled as is, all icons show (Xamarin.Forms 3.5.0.129452 & Iconize packages 3.5.0).
> * Upgraded Xamarin.Forms to 4.1.0.581479 all icons still show.
> * Upgraded all Iconize packages to 3.5.0.114 all icons still show
> * Upgraded all Iconize packages to latest 3.5.0.115 all icons **not** showing.
> 
> So it stopped working at Iconize version 3.5.0.115.

**Comment by mabolaji** [comment]
@chenzhenghong Your solution works for me. I have spend so much time checking for why the icons were not showing

**Comment by thisismyself** [comment]
Meanwhile, Iconize version 3.5.0.114 seems to be removed from Nuget.
